### ISSUE TITLE
Second

### ISSUE THREAD

**Issue by vickyantre** [Artinov/home-work-10-vickyantre#2]
commit: added disappearing X, cheanged checkbox
to easier find

1. When I click checkbox, checkbox has line-through too. It looks bad.
2. When I click checkbox text jump little higher and box getting smaller.

**Comment by vickyantre** [comment]
Also #todoList li button do not want to work with display:none;

I have to do color white.

**Comment by vickyantre** [comment]
Also I cant do border none to list-group. And I think it is a problem of list-group class bootstrap that When I click checkbox text jump little higher and box getting smaller.

But if i will remove list group, i`ll have to change js file, becouse I used it in js too.

**Comment by eko24ive** [comment]
Well here is what I discovered.
This "jump" happens because of chaning of `content` rule for `:before` pseudo-class.
By default checkbox don't have this check `✓` but when we add it - check symbol increases size of our `:before` element. This could be fixed by setting same check symbol for unchecked checkbox state and setting its opacity to 0. So we can't see it but it would have size equal with checked state.
Let me know if I explained this too vague.
